src/app/pages/main/store/epics/index.ts

```diff
@@ -0,0 +1,25 @@
+import { RootStore, Dependencies } from '@core/store';
+import { ItemsList } from '@core/pages/main/namespace';
+import { of } from 'rxjs';
+import { map, catchError, switchMap } from 'rxjs/operators';
+import { ofType, Epic } from 'redux-observable';
+
+import { MainActionTypes, loadItemsFailed, loadItemsSuccess } from '../actions';
+
+const executeGetItemsEpic: Epic<any, any, RootStore, Dependencies> = (
+  action$,
+  _state$,
```

Comment:
can we drop underscore prefixes?

Reply:
Underscore is forced by ts-lint. Otherwise I got a warning message that `state$` is declared but never used.